Telemetry/TelemetryEvent.swift

```diff
@@ -17,6 +17,8 @@ public class TelemetryEvent {
     public static let MaxNumberOfExtras = 10
     public static let MaxLengthExtraKey = 15
     public static let MaxLengthExtraValue = 80
+    
+    public static let ExtrasDefaultValue = "Extras"
```

Comment:
Hmm.. I think this should maybe be an empty string instead? `""`

Reply:
Whatever you both think is best. I'll wait for @garvankeeley to chime in too

Reply:
iI think empty string in case "Extras" is an actual value in future